modules/packages/LinearAlgebra.chpl

```diff
@@ -503,8 +503,44 @@ proc transpose(A: [?Dom] ?eltType) where isDenseMatrix(A) {
   }
 }
 
+pragma "no doc"
+proc tranpose(A: [?Dom] ?eltType, keepDistribution = true) 
+  where isDistributed(A) 
+{
+  // Add case for rank 1
```

Comment:
I believe this is a TODO? Are you hoping to do that in this PR, if not maybe add an error for now.